src/PresentationalComponents/Shared/ImageWithDefault.js

```diff
@@ -1,27 +1,15 @@
 import React from 'react';
 import propTypes from 'prop-types';
+import DefaultPortfolioImg from '../../assets/images/default-portfolio.jpg';
```

Comment:
> The ImageWithDefault is shared with other types of cards: PortfiolioItem, Platform and PlatformItem - so I would not use a 'Portfolio' named default image

I have removed default src form ImageWithDefault because it was not used anywhere. All components that are using ImageWithDefault have always set src attribute like this:
```JSX
src={Image || DefaultImage} defaultSrc={DefaultImage}
```
That is redundant. I understand what this component was supposed to do, but it was not actually doing anything because the image element did now have any reference (https://reactjs.org/docs/refs-and-the-dom.html), so you could not change to default image if the image load failed.

If we will need it in future we can create new component for that.

Reply:
Also we can get rid of the `ImageWithDefault` because its just wrapped `img` HTML tag. But i will do that in separate PR.

Reply:
The ImageWithDefault was added when we used image urls that may not exist or may not be reached - we are no longer using those now and is not clear if we will going forward, so we can remove this.